mods/ra/languages/lua/en.ftl

```diff
@@ -138,49 +138,49 @@ find-einstein-crashed-helicopter = Find Einstein's crashed helicopter.
 destroy-sam-sites = Destroy the SAM sites.
 hold-position-protect-base = Hold your position and protect the base.
 do-not-lose-more-than = Do not lose more than { $units } units.
-take-out-the-soviet-power-grid = Take out the Soviet power grid.
+take-out-the-soviet-power-grid = Disable the Soviet power grid.
 
 ## exodus
 units-evacuated = { $evacuated } / { $threshold } units evacuated.
 evacuate-units = Evacuate { $threshold } units.
 destroy-nearby-soviet-airbases = Destroy the nearby Soviet airbases.
-evacuate-at-least-one-gap-generator = Evacuate at least one mobile gap generator.
+evacuate-at-least-one-gap-generator = Save at least one Mobile Gap Generator.
 transferring-funds = Transferring funds.
 co-commander-missing = Co-Commander is missing
 
-## fall-of-greece-1-personal-war
+## fall-of-greece-01-personal-war
 civilians-manual-targeting = Commander, there are several civilians in the area.
-    We'll need you to call out targets.
-soviet-aircraft-armor-moving-in = Alfa Niner this is Lima One Six. Be advised, Soviet aircraft and armor moving into your AO.
+    We need you to designate targets.
+soviet-aircraft-armor-moving-in = Alfa Niner this is Lima One Six. Be advised, Soviet aircraft and armor are moving into your AO.
 headquarters = Headquarters
-extraction-point-compromised-evacuate-base = Extraction point is compromised. Evacuate the base!
-new-extraction-point-northwest = We've set up a new extraction point to the Northwest.
+extraction-point-compromised-evacuate-base = Extraction point compromised. Evacuate the base!
+new-extraction-point-northwest = New extraction point has been setup to the northwest.
 stalin-will-pay = Stalin will pay for what he has done today!
     I will bury him with my own hands!
 stavros = Stavros
 keep-stavros-tanya-alive = Keep Stavros and Tanya alive.
 get-tanya-stavros-helicopter = Get Stavros and Tanya to the extraction helicopter.
 
-## fall-of-greece-2-evacuation
-civilians-evacuated = { $evacuated } / { $threshold } civilians evacuated.
+## fall-of-greece-02-evacuation
```

Comment:
The original comment was the correct filename.

Reply:
It seems inconsistent compared to the numbering in other file names, but im happy to change it.